tools/appveyor.psm1

```diff
@@ -499,6 +509,15 @@ function Invoke-AppveyorFinish
         }
         Write-Log "MSI smoke test was successful"
 
+        # Verify path was updated by MSI
+        $psPath = ([System.Environment]::GetEnvironmentVariable('PATH','MACHINE')) -split ';' |
+            Where-Object {$_ -like '*files\powershell*' -and $_ -notin $beforePath}
```

Comment:
What if we'll always install in the same directory as we discussed in RFC?

Reply:
this pattern still matches